scss/components/_hero.scss

```diff
@@ -111,4 +110,26 @@
       @include shift(3);
     }
   }   
+}
+
+.hero__banner--monument {
+  background-image: url('../img/monument.jpg');
+}
+
+.hero__banner--office {
+  display: none;
+
+  @include media($med) {
+    display: block;
```

Comment:
Properties should be ordered background-image, background-position, background-repeat, background-size, display, height